homestead/after.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+# If you would like to do some extra provisioning you may
+# add any commands you wish to this file and they will
+# be run after the Homestead machine is provisioned.
+#
+# If you have user-specific configurations you would like
+# to apply, you may also create user-customizations.sh,
+# which will be run after this script.
+
+cd /home/vagrant/code
+
+sudo cp /home/vagrant/code/homestead/php.ini /etc/php/7.2/mods-available/custom.ini
+sudo phpenmod -v 7.2 custom
+composer install
+xon
+echo 'cd /home/vagrant/code' >> ~/.profile
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+
+nvm install 8.9.1
```

Comment:
This needs to be set to a Node 10 version to get the dev env to work as described in the development readme.